<?xml version="1.0"?>

<launch>

 <!-- Launch elevation mapping node. -->
  <node pkg="elevation_mapping" type="elevation_mapping" name="elevation_mapping" output="screen">
    <rosparam command="load" file="$(find rossim)/config/rambler_robot.yaml"/>
    <rosparam command="load" file="$(find rossim)/config/postprocessor_pipeline.yaml"/>
  </node>

</launch>
  
  
